reader/tests/system/exists/test_exists.py

```diff
@@ -34,7 +35,16 @@
 
 
 def setup_data(connection, cursor, deleted=False):
-    for fqid, model in list(data.items()) + list(other_models.items()):
```

Comment:
Please deduplicate the code. Maybe extract this mehtod and pass (data, other_models) as additional params.

Reply:
Done.